053_main_L28/053_main_L28_safe.py

```
@@ -0,0 +1,194 @@
+            'numpy': {'ndarray', 'dtype', 'int64', 'float64', 'int32', 'float32'},
+            'torch': {'Tensor', 'Size', 'dtype', 'device'},
+            'collections': {'OrderedDict', 'defaultdict', 'Counter'},
+            '__main__': set()  # Allow main module classes if needed
+        }
+        
+        if module in safe_modules:
+            if name in safe_modules[module] or module == '__main__':
+                return super().find_class(module, name)
+        
+        # Block everything else to prevent code injection
+        raise pickle.UnpicklingError(f"Blocked unsafe class: {module}.{name}")
```

Ruff (0.14.1)

Avoid specifying long messages outside the exception class

(TRY003)